Z/zbc2014/build_tarballs.jl

```diff
@@ -0,0 +1,26 @@
+using BinaryBuilder
+
+name = "zbc2014"
```

Comment:
Though the name is sufficiently random, that nobody would likely ever need this for anything more general, `zbc2014` still seems like an overly terse name for something very specific (the implementation for a scientific publication in a certain academic field).

```suggestion
name = "ZilanyBruceCarney2014"
```

Notably, to match https://github.com/guestdaniel/ZilanyBruceCarney2014.jl

Reply:
I have no strong preference regarding the name being abbreviated or not. Happy to do what is advised. My only hesitation is that I glanced at a few of the other folders in Yggdrasil and did not any other packages using last names. Let me know what is advised here.